client.go

```diff
@@ -200,7 +203,7 @@ type tradingStatusRequest struct {
 	Symbols []string `url:"symbols,comma,omitempty"`
 }
 
-// The Exchange may suspend trading of one or more securities on IEX
```

Comment:
If you have a chance, can you update these to read as full sentences like the others?